src/librustdoc/clean/types.rs

```diff
@@ -1090,6 +1100,18 @@ impl AttributesExt for Attributes {
     fn lists(&self, name: Symbol) -> ListAttributesIter<'_> {
         self.other_attrs.lists(name)
     }
+    
+    fn span(&self) -> Option<rustc_span::Span> {
+        todo!("delete");
+    }
+    
+    fn inner_docs(&self) -> bool {
+        todo!("delete");
+    }
+    
+    fn other_attrs(&self) -> Vec<ast::Attribute> {
+        self.other_attrs.clone()
+    }
```

Comment:
Rather than add functions that aren't supported, do you think it makes sense *not* to implement AttributesExt for Attributes, and add `lists` and `other_attrs` as inherent functions? That would give a compile error if you tried to call an unsupported functoin.